jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/assignment/impl/LoadCalculatorImpl.java

```diff
@@ -0,0 +1,16 @@
+package org.jbpm.services.task.assignment.impl;
+
+import org.jbpm.services.task.assignment.LoadCalculator;
+
+public abstract class LoadCalculatorImpl implements LoadCalculator {
```

Comment:
a minor comment ... maybe rename this class to AbstractLoadCalculator instead of LoadCalculatorImpl as (at least to me) it indicates actual impl